src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -430,36 +265,15 @@ impl HummockManager {
             );
         }
 
+        drop(versioning_guard);
         tracing::trace!("new committed epoch {}", epoch);
 
-        let mut table_groups = HashMap::<u32, usize>::default();
-        for (table_id, info) in versioning.current_version.state_table_info.info() {
-            table_groups.insert(
-                table_id.table_id,
-                versioning
-                    .current_version
-                    .state_table_info
-                    .compaction_group_member_tables()
-                    .get(&info.compaction_group_id)
-                    .expect("should exist")
-                    .len(),
-            );
-        }
-        drop(versioning_guard);
         // Don't trigger compactions if we enable deterministic compaction
         if !self.env.opts.compaction_deterministic_test {
             // commit_epoch may contains SSTs from any compaction group
             for id in &modified_compaction_groups {
                 self.try_send_compaction_request(*id, compact_task::TaskType::Dynamic);
             }
-            if !table_stats_change.is_empty() {
```

Comment:
Where will we do this stats cleaning logic after this PR?

Reply:
purge_prost_table_stats